### DISCUSSION TITLE
Add RT-A/B-flag

### DISCUSSION THREAD

**Discussion by lxsxl** [PE5PVB/TEF6686_ESP32#281] [Ideas]
If I remember correctly there somewhen was a request on Discord to add RT-A/B-flag. This can be useful as sometimes the flag toggles while RT-content remains the same. Hopefully this can be added easily to main screen and/or advanced RDS-screen?

**Comment by PE5PVB** [comment]
I can do that, but where to put it? Screen is already crowded.

**Comment by andimik** [comment]
I think two dots are enough, like `RT :`, the upper for A and the lower for B. For example mark it in a color, but I doubt you will identify this tiny dot.

This will not break the layout, BTW.

**Comment by lxsxl** [comment]
I agree with Andis idea (two dots), a green dot for the currently displayed text, the other one in red.

Btw: Could you please again extend the RT-scrolling for about 5 characters (only on advanced RDS-screen)? In case of very long texts it is still cut off a bit too early.

Thx!

**Comment by lxsxl** [comment]
RT flag A/B could probably be shown like this? Maybe even as a replacement of the white colon within "RT:"? And same on advanced RDS screen:

![RT Flag A-B](https://github.com/PE5PVB/TEF6686_ESP32/assets/140750201/05cea14b-da04-4815-8ff1-e8dcfcb11b67)

**Comment by PE5PVB** [comment]
Will be implemented

**Reply by lxsxl** [reply]
Perfect, thanks!

**Comment by PE5PVB** [comment]
AB flag implemented.
